templates/app/client/components/auth(auth)/router.decorator.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-import _ from 'lodash';
+import * as _ from 'lodash';
```

Comment:
Well yes, I believe if there is a default export, there should be no difference. Lodash does however not appear to have one. Visual studio code warned me about that particular fact and I think it was among the typings error spam. 

As far as I've read elsewhere you need to import it like above when there is no default export. Or  use a require.

Before the above fix I also get the following error in the browser when changing route. I.e. going from home to admin or settings:
```javascript
angular.js:13708 TypeError: Cannot read property 'noop' of undefined
    at router.decorator.ts:25
    at Scope.$broadcast (angular.js:17767)
    at Object.transitionTo (angular-ui-router.js:3272)
    at Object.go (angular-ui-router.js:3107)
    at angular-ui-router.js:4143
    at angular.js:19374
    at completeOutstandingRequest (angular.js:5955)
    at angular.js:6234
```
After the change the error does not appear again. Failing checks have me nervous but tests seem to pass with "gulp test" locally for me at least. Though there was a too long timeout (5000+ ms) on the server 1 out of 2 times.